bundles/CoreBundle/src/Migrations/Version20230107224432.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230107224432 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Grant XLIFF import/export permission to those who already had "translation" permission, due XLIFF functionalities being moved out to be Standalone from Core Translation';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // Append to the comma separated list whenever the permissions text field has 'translation' but not already xliff_import_export
+        $this->addSql(
+            'UPDATE users SET permissions = CONCAT(permissions, \',xliff_import_export\')
+            WHERE permissions LIKE \'%translation%\' AND permissions NOT LIKE \'%xliff_import_export%\''
```

Comment:
mhhh, \'%translation%\' could be too generic, might be falsely triggered in something like `somebundle_translation`